alacritty_terminal/src/term/mod.rs

```diff
@@ -348,24 +348,25 @@ impl<T> Term<T> {
     /// Convert the active selection to a String.
     pub fn selection_to_string(&self) -> Option<String> {
```

Comment:
Why did you decide to change the `selection_to_string` behavior now?

Reply:
The reason is that keep lines selection behavior even if the behaviors of simple/semantics are changed.

> This works, no need to re-design Alacritty's selection copying when that part of the code clearly worked just fine.

I interpreted it as no need to change not related selection behavior to hint action, but It makes sense that `semantics` selection's behavior also is changed because it is very similar to `simple` selection, which is related to `select` hint action.

`block` selection's behavior is also changed in this commit, but it is my mistake. I will revert to previous version.

Reply:
I don't mind the new behavior, just really confused me after my last comment.